Topics/User_Experience/Navigation design in complex websites.md

```diff
@@ -0,0 +1,72 @@
+     * For example, home page to sign in back to home page
+     * Additihnally, consider using prototyping or wireframing to give yourself a clearer picture on how the user may naviage through your website
+       
+3. Determine which pages the user may need to reference most often
+    * For example, the home page / landing page is usually referenced a lot by the user.
+        
+4. Consider using some of the following to ease navigation through the website :  
+    * Navbars
+        * Usually at the top of a landing page / home page. These contain links to the website's main pages. 
+    * Side menus
+        * Usually located at the side of the page and contains links to relevant pages on the website 
+    * Tabbed menus
+        * Organizes information into different tabs, where each tab contains different relevant information
+    * Footers containing links to all relevant pages within your website 
+        * Usually at the bottom of a page. It can contain links to all pages on the website as well as contact information.
+          
+5. Consider how the user may navigate through the content on a single page.
+   * Sub-navbars can be utilized.
+        * Located below a normal Navbar and  consists of links that are relevant to the specific page that the user is on 
+    *  If there is searchable / filterable content on this page, consider using icons that are easily identifiable by the user as  search  ( magnifying glass) or filter. 
+          
+* Please see the link below for more navigation design patterns:
+    * [https://ui-patterns.com/patterns/navigation/list](https://ui-patterns.com/patterns/navigation/list)
+      
+
+## Advantages of Good Navigation Design for Complex Websites  
+
+* Good navigation design allows a clear or intuitive path for users to navigate through the web app / website  
+    * For example, an easy/ clear path through a website to specific content, may encourage users to use your product.
+* Increases usability 
+    * A good navigation design allows users to easily find the content that they need in a timely manner . 
+* Increases user satisfaction 
+    * Users are less likely to become frustrated navigating a website or webapp if there is a clear navigation system.  
+     
+
+
+## Key Things to Consider for Navigation Design of Complex Websites 
+* Accessibility 
+    * Additional tips on designing accessible menus/ navigation.
+        * [Click Here](https://www.w3.org/WAI/tutorials/menus/)
+* Important links should be kept in navigation menus / nav bars
+* Different screen sizes may require varying navigation designs 
+    * For example, desktops  with navbars vs mobile devices with sidebar menus 
+* Is it easy to navigate through the main pages of my website ?
+    * A good way to determine this, is to run through your website before project completion. Navigate the website as a user while looking for any pitfalls that users may notice but you did not as a developer. 
```

Comment:
Thank you for adding this based off my suggestion! It really helps by giving students a starting point to answer this question.

Reply:
Thank you for the suggestion ! I agree, it really helps to give you a great idea on where you can start to incorporate good navigation design.